### PR TITLE
Implemented Structure

### PR DESCRIPTION
**Pull request by Iyanu-Kit**

### CODE REVIEW

**Comment by Iyanu-Kit** [comment]
It is ready for review @borngraced

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+    <head lang="en">
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equip="X-UA-Compatible" content="ie=edge">
+        <title>Project-structure</title>
```
Comment (borngraced) [inline index.html:7]: title should be the title of the project you're creating

there should be `description` after `title` too
```javascript
<meta name="description" content="project description"/>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        <link rel="stylesheet" href="main.css">
+        <body>
+            <script src="app.js"></script>
+        </body>
+    </head>
+</html>
```
Comment (borngraced) [inline index.html:13]: add a line break break at the end of the file (press enter)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    <head lang="en">
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equip="X-UA-Compatible" content="ie=edge">
+        <title>Project-structure</title>
+        <link rel="stylesheet" href="main.css">
+        <body>
+            <script src="app.js"></script>
+        </body>
+    </head>
```
Comment (borngraced) [inline index.html:12]: what's is `</head>` doing at the bottom? please check reference again to see how the boiler template is done

**Review by borngraced — COMMENTED** [review]
one more note.

where's footer and header??
please read more about them here [https://www.w3schools.com/tags/tag_header.asp](https://www.w3schools.com/tags/tag_header.asp) and [https://www.w3schools.com/tags/tag_footer.asp](https://www.w3schools.com/tags/tag_footer.asp)
```html
<!DOCTYPE html>
 <html>
     <head lang="en">
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <meta http-equip="X-UA-Compatible" content="ie=edge">
         <title> Portfolio </title>
         <meta name="description" content="My-Portfolio"/>
         <link rel="stylesheet" href="main.css">
       <header> </header>
         <body>
             <script src="app.js"></script>
         </body>
         <footer> </footer>
     </head>
```

**Comment by Iyanu-Kit** [comment]
I have corrected the <head> tag and added the header and footer.

**Comment by borngraced** [comment]
LGTM!

**Pull request merged**